### PR TITLE
Fixed issue where generic fallback methods wouldn't work for methods with many parameters (#1174)

### PR DESCRIPTION
**Pull request by ArvinJA**
Fix for https://github.com/resilience4j/resilience4j/issues/1174

### CODE REVIEW

**Comment by hexmind** [comment]
Could you add unit test also?

**Comment by vanadzins** [comment]
> Could you add unit test also?

@hexmind is the added unit test enough?

File: resilience4j-spring/src/test/java/io/github/resilience4j/fallback/FallbackMethodTest.java [hunk 1]
```diff
@@ -56,6 +56,16 @@ public void fallbackGlobalExceptionWithSameMethodReturnType() throws Throwable {
             .isEqualTo("recovered-IllegalStateException");
     }
 
+    @Test
+    public void fallbackGlobalExceptionWithSameMethodReturnTypeAndMultipleParameters() throws Throwable {
+        FallbackMethodTest target = new FallbackMethodTest();
+        Method testMethod = target.getClass().getMethod("multipleParameterTestMethod", String.class, String.class);
+        FallbackMethod fallbackMethod = FallbackMethod
+            .create("fallbackMethod", testMethod, new Object[]{"test", "test"}, target);
+        assertThat(fallbackMethod.fallback(new IllegalStateException("err")))
```
Comment (hexmind) [inline resilience4j-spring/src/test/java/io/github/resilience4j/fallback/FallbackMethodTest.java:65]: just cosmetic/convention changes (empty lines):

```
FallbackMethod fallbackMethod = FallbackMethod
            .create("fallbackMethod", testMethod, new Object[]{"test", "test"}, target);

String fallback = fallbackMethod.fallback(new IllegalStateException("err");

 assertThat(fallback)).isEqualTo("recovered-IllegalStateException");
```

**Review by hexmind — CHANGES_REQUESTED** [review]
LGTM
Did you also run a test with a previous implementation (`args.length == 1`) to catch a error?

File: resilience4j-spring/src/test/java/io/github/resilience4j/fallback/FallbackMethodTest.java (hunk 1 above)
Comment (vanadzins) [reply resilience4j-spring/src/test/java/io/github/resilience4j/fallback/FallbackMethodTest.java:67]: Added the empty lines. Also I had it tested with the previous implementation to see that the test fails 🙂

**Review by hexmind — COMMENTED** [review]
Thank for the changes.
@RobWin is OK?

**Review by RobWin — APPROVED** [review]

**Comment by RobWin** [comment]
thx

**Comment by alesanderlg** [comment]
Hey @RobWin, 

I am facing this problem in the current version 1.5.0. I saw you guys have fixed it. Do you guys have any plans to release this fixed for now?

In case you have got it done, what's the new release version?

Thanks in advance.

**Pull request merged**